design/nations/nation-validators.js

```diff
@@ -5,6 +5,24 @@ function (newDoc, oldDoc, userCtx, secObj) {
       throw { forbidden: errMessage };
     }
   }
+  function canWrite() {
+    var CAN_WRITE = true;
+    var readOnly = ['learner'];
+    if (!userCtx.isUserAdmin) {
+      if (userCtx.roles.length < 1) {
+        CAN_WRITE = false;
+      } else {
+        for (var i = 0; i < userCtx.roles.length; i++) {
+          if (~ readOnly.indexOf(userCtx.roles[i])) {
+            CAN_WRITE = false;
+          }
+        }
+      }
+    }
+    if(!CAN_WRITE)
```

Comment:
Even for a one line block, it's better (more readable, doesn't risk someone writing another line & popping that statement out of the if clause) to have brackets.

Reply:
oops yes i had noticed it but forgot to push that change :) thanks for catching